docs/keycloak/manual-client-secret-setup-pl.md

```
@@ -0,0 +1,92 @@
+3. Przejdź do sekcji **Clients** (klienci) z lewego menu.
+4. Wybierz klienta typu confidential, którego konfigurację chcesz uzupełnić. ![Manage clients](./images/manage-clients.png)
+5. Przejdź do zakładki **Credentials**. ![Client credentials](./images/client-credentials.png)
+6. Wygeneruj nowe poświadczenie klikając **Regenerate secret**. ![Regenerate client secret](./images/regenerate-client-secret.png)
+7. Skopiuj wartość `client_secret` i użyj jej w konfiguracji swojej aplikacji.
+
+## 🧩 Konfiguracja aplikacji do współpracy z lokalnym Keycloak
+
+Aby aplikacja mogła poprawnie komunikować się z lokalnie działającą instancją Keycloak, należy skonfigurować w niej odpowiednie wartości połączeniowe.
+
+W obecnym stanie projektu:
+- Środowisko **`prod`** nie posiada jeszcze kompletnej infrastruktury umożliwiającej działanie.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)